modules/appendix/examples/ldbc_snb/select_statement_accum.gsql

```diff
@@ -3,9 +3,9 @@ CREATE QUERY accum_wrong_example() SYNTAX v2 {
     SumAccum<INT> @active_flag = 0;
 
     result = SELECT p
-             FROM Person: p - (KNOWS) - Person: w
+             FROM (p:Person) -[KNOWS]-> (w:Person)
              WHERE w.lastName == "Wang" AND p.firstName == "Peter"
              ACCUM p.@active_flag += 1,
-                   @@count_total += p.@active_flag; // <1>
+                   @@count_total += p.@active_flag;
```

Comment:
```suggestion
                   @@count_total += p.@active_flag; // <1>
```
Why did you remove this?  This is a **callout**, which was serving a purpose:
https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/